internal/tui/nomad/pages.go

```diff
@@ -314,6 +367,12 @@ func namespaceFilterPrefix(namespace string) string {
 
 func (p Page) GetFilterPrefix(namespace, jobID, taskName, allocName, allocID string, eventTopics Topics, eventNamespace string) string {
 	switch p {
+	case ConfirmPage:
+		return "🔴 Are you sure?"
```

Comment:
I've found emojis to have weird width in terminals, e.g. https://github.com/charmbracelet/lipgloss/issues/55. Let's remove this one